src/components/pages/EdcConnector/edcConnectorTableColumns.tsx

```diff
@@ -116,10 +106,35 @@ export const ConnectorTableColumns = (
         </>
       ),
     },
+    {
+      field: 'connectorUrl',
+      headerName: t('content.edcconnector.columns.connectorUrl'),
+      flex: 1,
+      sortable: false,
+      disableColumnMenu: true,
+      align: 'center',
+      headerAlign: 'center',
+    },
     {
       field: 'details',
       headerName: t('content.edcconnector.columns.details'),
-      flex: 1.2,
+      flex: 0.8,
+      sortable: false,
+      disableColumnMenu: true,
+      align: 'center',
+      headerAlign: 'center',
+      renderCell: () => {
+        return (
+          <IconButton color="secondary">
+            <ArrowForwardIcon />
+          </IconButton>
+        )
+      },
```

Comment:
guess we can write that more compact (formatting ignored here):

`renderCell: () => <IconButton color="secondary"><ArrowForwardIcon /></IconButton>`